lib/large_text_field/owner.rb

```diff
@@ -61,21 +69,29 @@ def text_field_changed(field_name)
       text_field_hash_loaded? && @text_field_hash[field_name]&.changes&.any?
     end
 
+    # rubocop:disable Metrics/CyclomaticComplexity
     def validate_large_text_fields
       if text_field_hash_loaded?
-        large_text_field_options.each do |k, options|
-          value = text_field_hash[k]&.value
+        large_text_field_options.each do |key, options|
+          value = text_field_hash[key]&.value
           conjugation = options[:singularize_errors] ? "is" : "are"
           maximum = options[:maximum] || MAX_LENGTH
-          errors.add k, "#{conjugation} too long (maximum is #{self.class.formatted_integer_value(maximum)} characters)" if value.present? && value.size > maximum
+
+          next unless value.present? && value.size > maximum
+
+          errors.add(
+            key,
+            "#{conjugation} too long (maximum is #{self.class.formatted_integer_value(maximum)} characters)"
+          )
```

Comment:
⛏️ I typically find early returns or nexts to be more confusion to grok (had to read this a few times to understand what's going on. Rather than the early return, could we just wrap the `errors.add` in the if statement instead of the trailing if?